azure/durable_functions/models/actions/CallEntityAction.py

```diff
@@ -16,13 +16,14 @@ class CallEntityAction(Action):
 
     def __init__(self, entity_id: EntityId, operation: str, input_=None):
         self.entity_id: EntityId = entity_id
-        self.instance_id: str = EntityId.get_scheduler_id(entity_id)
-        self.operation: str = operation
-        self.input_: str = dumps(input_, default=_serialize_custom_object)
 
         if not self.entity_id:
             raise ValueError("entity_id cannot be empty")
 
+        self.instance_id: str = EntityId.get_scheduler_id(entity_id)
+        self.operation: str = operation
+        self.input_: str = dumps(input_, default=_serialize_custom_object)
+
```

Comment:
Somehow GitHub is not reflecting this change, so I'm highlighting this for visibility. We check if entity_id is `None` before trying to get the instance_id.